concordium-consensus/src/Concordium/GlobalState/Persistent/BlobStore.hs

```diff
@@ -914,6 +914,12 @@ class Monad m => Reference m ref a where
     -- it will just be stored in memory as cached.
     refMake :: a -> m (ref a)
 
+    -- |Create a reference to a value and flush it. It returns the flushed reference.
+    refMakeFlushed :: a -> m (ref a)
+    refMakeFlushed v = do
```

Comment:
I'd make this strict in `v`
```suggestion
    refMakeFlushed !v = do
```

Reply:
Although it probably does not matter since the value is flushed immediately, as long as the result of flushing is used.